Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelWizard/AddAPIModelWizard.cs

```diff
@@ -68,55 +71,82 @@ public enum eAPITypeTemp
         public override string Title { get { return "API Model Import Wizard"; } }
 
         public RepositoryItemKey TargetApplicationKey { get; set; }
-        
+
         public ObservableList<RepositoryItemKey> TagsKeys = new ObservableList<RepositoryItemKey>();
 
         public bool AvoidDuplicatesNodes { get; set; }
 
         public AddAPIModelWizard(RepositoryFolder<ApplicationAPIModel> APIModelsFolder)
         {
             APIModelFolder = APIModelsFolder;
-            
+
             AddPage(Name: "Introduction", Title: "Introduction", SubTitle: "API Model Introduction", Page: new WizardIntroPage("/ApplicationModelsLib/APIModels/APIModelWizard/AddAPIModelIntro.md"));
 
             AddPage(Name: "Select Document", Title: "API's Import Source", SubTitle: "Set API's Import Source/s", Page: new AddAPIModelSelectTypePage());
 
             AddPage(Name: "Select API", Title: "Select API's", SubTitle: "Select Desired API's to Add", Page: new ScanAPIModelWizardPage());
 
             AddPage(Name: "Learn Optional Parameters Values", Title: "Learn Optional Parameters Values", SubTitle: "Learn Optional Parameters Values from Sample Request Files", Page: new AdAPIModelMappingPage());
-            
+
             AddPage(Name: "Extra Configurations", Title: "API's Extra Configurations", SubTitle: "Set API's Extra Configurations", Page: new AddAPIModelExtraConfigsPage());
         }
 
         public override void Finish()
         {
             //ExportAPIFiles(SelectedAAMList);
-            ExportAPIFiles(General.ConvertListToObservableList(AAMList.Where(x => x.IsSelected == true).ToList()));
+            if(DeltaModelsList != null && DeltaModelsList.Count > 0)
+            {
+                foreach(DeltaAPIModel deltaAPI in DeltaModelsList.GroupBy(d => d.matchingAPIModel).Select(d => d.First()))     // (DeltaAPIModel.matchingAPIModel)))          //.Where(d => d.IsSelected))
+                {
+                    if(deltaAPI.DefaultOperationEnum == DeltaAPIModel.eHandlingOperations.MergeChanges 
```

Comment:
why field called "DefaultOperationEnum" while it is the selected operation done by user, please rename it to "SelectedOperation"

Reply:
Updated. Done !